### DISCUSSION TITLE
feature: Live server configuration editing via editor

### DISCUSSION THREAD

**Discussion by WeylonSantana** [AscensionGameDev/Intersect-Engine#2712] [Ideas]
### Description

In the near future (between 1 and 50 years) the intersect interface will change, perhaps to the end of the 50 million json files in the gui. I thought there might be the same for the server. In "file" could have an option type, preferences, server config or etc, where we could change the colors and config checks there, by the editor.

### Intended Use-Case

Configure server options, from the editor, without needing to restart the server for every modified line in the json (if possible)

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by pandinocoder** [comment]
This will be exposed in the Toolkit when I am done (though it won't be live).

Live editing can be added, but will not be complete. Some things like map/tile sizes can't be changed while running.
